test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
-});
\ No newline at end of file
+  it("returns a reversed array of the number's digits", () => {
```

Comment:
Could some more tests be added here to make sure it works for arrays of different lengths?